backend/app/src/configs/chat-event.constants.ts

```diff
@@ -1,7 +1,10 @@
 export const chatEvent = {
   JOIN: 'JOIN',
   LEAVE: 'LEAVE',
+  CREATE: 'CREATE',
   SEND: 'SEND',
   RECEIVE: 'RECEIVE',
   NOTICE: 'NOTICE',
+  ADD_ADMIN: 'ADD_ADMIN',
+  REMOVE_ADMIN: 'REMOVE_ADMIN',
 }
```

Comment:
```suggestion
  JOIN,
  LEAVE,
  CREATE,
  SEND,
  RECEIVE,
  NOTICE,
  ADD_ADMIN,
  REMOVE_ADMIN,
} as const
```
[es2015 객체 표기법 참고링크](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Object_initializer#ecmascript_2015의_새로운_표기법)
[타입 단언 (type assertion) 참고링크](https://blog.toycrane.xyz/typescript%EC%97%90%EC%84%9C-%ED%9A%A8%EA%B3%BC%EC%A0%81%EC%9C%BC%EB%A1%9C-%EC%83%81%EC%88%98-%EA%B4%80%EB%A6%AC%ED%95%98%EA%B8%B0-e926db079f9)